media/transcoder/transcoder_test.go

```diff
@@ -99,6 +100,8 @@ func TestJobTemplatesAndJobs(t *testing.T) {
 	t.Logf("\nwriteTestGCSFiles() completed\n")
 	testJobFromPreset(t, projectNumber, inputURI, outputURIForPreset)
 	t.Logf("\ntestJobFromPreset() completed\n")
+	testJobFromPresetBatchMode(t, projectNumber, inputURI, outputURIForPresetBatchMode)
+	t.Logf("\ntestJobFromPresetBatchMode() completed\n")
```

Comment:
Don't have to change it in this PR but it would probably be nicer to use subtests or fully separated tests rather than these test functions.

Reply:
I looked into refactoring this, but the `testutil/storage_cleaner.go` functions take a parameter of `t *testing.T`. As far as I can tell, this parameter isn't available in a `TestMain(m *testing.M)` function. I need to do some GCS bucket setup in `TestMain()`.